README.md

```diff
@@ -61,6 +61,47 @@ If you want your mail server to scan messages for viruses and spam, you will hav
 # apt-get install clamav-daemon clamav-freshclam spamassassin
 ```
 
+**EL:** - For RedHat based (Enterprise Linux, CentOS, RHEL, Scientific) Exim is not build linked against Mysql libraries making it imposible to use vexim against a RDBMS such as mysql or postfix. The EPEL project provides support for Exim that will utilize Mysql. The EPEL repo is a Red Hat sponsored community based repo, details can be found https://fedoraproject.org/wiki/EPEL
+
+We first must enable the EPEL repo for the right version of Enterprise Linux. Your host will need outgoing access to the internet on port 443.
+for Enterprise Linux 6.X
+```
+rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
+yum update
+``` 
+For Enterprise Linux 7.X
+```
+rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+yum update
+```
+With the correct EPEL repo enabled, you can now install the correct exim package from epel.
+```
+yum install exim-mysql
+```
+This package will pull in exim with Mysql support and associated dependencies.
+```
```

Comment:
These backticks aren't necessary.
For the reference: backticks denote source code or shell commands, so you should only add them where necessary.